openprescribing/templates/all_pcns.html

```diff
@@ -6,12 +6,25 @@
 
 {% block content %}
 
-<h1>All PCNs</h1>
+<h1>Find a PCN</h1>
 
+<p>Search for a Primary Care Network by name or ONS code</p>
```

Comment:
Are they ONS codes or ODS codes?

Reply:
I _think_ they're ONS codes. My understanding is that we prefer ODS codes and use the field `code` for those, but where they are not currently available (as with STPs and most likely PCNs) we use ONS codes with the field `ons_code`.

Reply:
We don't know in this case. There are neither ODS nor ONS codes available, currently. Perhaps just "...by name or code"?!